apps/src/calc/calc.js

```diff
@@ -80,6 +81,12 @@ function getTokenList(one, two) {
   if (two instanceof Equation) {
     two = two.expression;
   }
+  if (jsnums.isSchemeNumber(one)) {
+    one = new ExpressionNode(one);
+  }
+  if (jsnums.isSchemeNumber(two)) {
+    two = new ExpressionNode(two);
+  }
   if (typeof(one) === 'string') {
```

Comment:
Re: the previous comment, maybe I'm confused about is what `one` being a string and what `marked` indicate outside of the context of this function since I just jumped into it without context :)

Is this interpretation correct: `one` being a string means it's a variable name? And `marked` means `markedAsIncorrect`?